farrow/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!-- wp:template-part {"slug":"header","tagName":"header","layout":{"inherit":true}} /-->
```

Comment:
It looks like the only difference between this template and the one in Blockbase is that the header inherits the default layout. My preference would be to move that inside the header itself (creating a new header which inherits layout). This would give users more header template options, and limit unnecessary variations to our templates.